src/components/FilterGroup.tsx

```diff
@@ -94,6 +104,7 @@ export function FilterGroup({
             />
           );
         })}
+        {limited && <button className='text-blue-500 py-1 text-sm' onClick={toggleShowAll}>{showAll ? 'Show Less' : 'Show More'}</button>}
```

Comment:
instead of text-blue-500, let's update it to use the corresponding custom `primary` color

Reply:
looks like that failed the percy build, could you approve it?